conda/recipes/cuml/build.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 
 # This assumes the script is executed from the root of the repo directory
-./build.sh cuml
+./build.sh cuml $SINGLE_GPU
```

Comment:
Where is `SINGLE_GPU` defined? Is this an env var defined on a Jetson?

Reply:
I expected it to be defined at the command line, the way I was building was `CUDA=10.2 SINGLE_GPU=--singlegpu conda build conda/recipe/libcuml`.  It worked great for me.  It doesn't exist naturally, the user will have to flag it somehow.